### ISSUE TITLE
Coverage includes dependent packages when using a local path.

### ISSUE THREAD

**Issue by Morgan2010** [mattpolzin/swift-codecov-action#4]
The coverage includes dependent packages when those dependencies are created using local paths.

Example Package.swift:
```swift
let package = Package(
    ...
    dependencies: [
        .package(path: "../Dependency1"),
        .package(name: "Dependency2", path: "../Dependency2")
    ],
    ...
)
```

**Comment by mattpolzin** [comment]
This should be fixed with version [0.7.2](https://github.com/mattpolzin/swift-codecov-action/releases/tag/0.7.2) of this action.

**Comment by mattpolzin** [comment]
Nevermind, the solution works for standalone execution, but I see now I missed an important bit when I forgot that this action gets run inside a Docker container.

**Comment by mattpolzin** [comment]
Ok, _now_ this appears to be addressed with version [0.7.3](https://github.com/mattpolzin/swift-codecov-action/releases/tag/0.7.3). You will need to specify a `PROJECT_NAME` input for the GitHub action and set that input to the exact spelling of the root folder of your target project. After that, the coverage numbers should correctly filter out dependencies created with the `path` package initializers.

Let me know if this isn't working for you!
